src/solvers/convex_bundle_method.jl

```diff
@@ -399,6 +333,9 @@ function convex_bundle_method!(
     vector_transport_method::VTransp=default_vector_transport_method(M, typeof(p)),
     sub_problem=convex_bundle_method_subsolver,
     sub_state=evaluation,
+    k_size=nothing,
```

Comment:
Could you add to those that are deprecated a comment `# deprecated` (every line affected) so it is easier to remove that when we do a breaking release? That would be great.